src/layouts/MyLayout.vue

```diff
@@ -4,110 +4,116 @@
       elevated
     >
       <q-toolbar>
+        <q-toolbar-title>
+          <font
+            face="impact"
+            size="6"
+          >
+            ICEMEME
+          </font>
+        </q-toolbar-title>
+
+        <q-btn
+          color="secondary"
+          :to="{ name: 'register' }"
+        >
+          Sign Up / Log In
+        </q-btn>
```

Comment:
Because sign up is its own page and login is handled by a dialog (or maybe a dropdown), lets use two separate buttons here.

Also, replace `color="secondary"` with `flat` and `stretch` to make toolbar-style buttons

Reply:
What I had in mind is instead of having two buttons the one button would open the login dialog which would have a link to the sign-up page

Reply:
That could also work, I'm just used to seeing both buttons separate. It would also make for fewer clicks to sign up.

Your choice. I'd recommend looking at how other sites do it